manifest_staging/charts/aad-pod-identity/values.yaml

```diff
@@ -232,7 +232,7 @@ nmi:
 
   # enable running aad-pod-identity on clusters with kubenet
   # default is false
-  allowNetworkPluginKubenet: false
+  allowNetworkPluginKubenet: true
```

Comment:
Was this an change intended?

Reply:
Good catch. I turned this on for testing but forgot to revert the change